client/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -1,5 +1,73 @@
-import { FC } from 'react';
-import { ProductCard } from '@/components/ProductCard/ProductCard';
-export const ProductPage: FC = () => {
-  return <ProductCard />;
+import style from './styles.module.scss';
+import img1 from '@/assets/product_p.png';
+import button_up from '@/assets/button_up.svg';
+import button_down from '@/assets/button_down.svg';
+import { ProductsDescription } from '@/components/ProductDescription/ProductsDescription';
+import { BottomDescription } from '@/components/BottomDescription/BottomDescription';
+import { SectionTitle } from '@/shared/components/SectionTitle/SectionTitle';
+import { Products } from '@/components/Products/Products';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from '@/app/store';
+import { useParams } from 'react-router-dom';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { addToCart } from '@/features/cart/cartSlice';
+
+export const ProductPage = () => {
+  const id: number = Number(useParams().id);
```

Comment:
it is better to do this way for readablity

const { id: idString } = useParams()

cosnt id = Number(idString)